fastlane/lib/fastlane/cli_tools_distributor.rb

```diff
@@ -15,24 +15,27 @@ def running_help_command?
       def take_off
         before_import_time = Time.now
 
-        # Usually in the fastlane code base we use
-        #
-        #   Helper.show_loading_indicator
-        #   longer_taking_task_here
-        #   Helper.hide_loading_indicator
-        #
-        # but in this case we haven't required FastlaneCore yet
-        # so we'll have to access the raw API for now
-        require "tty-spinner"
-        require_fastlane_spinner = TTY::Spinner.new("[:spinner] 🚀 ", format: :dots)
-        require_fastlane_spinner.auto_spin
-
-        # this might take a long time if there is no Gemfile :(
-        # That's why we show the loading indicator here also
-        require "fastlane"
-
-        require_fastlane_spinner.success
+        if !FastlaneCore::Env.truthy?("FASTLANE_DISABLE_ANIMATION")
```

Comment:
i think this is a chicken-egg problem, we haven't loaded fastlane_core at this point-in-code - however you are using `FastlaneCore::`- so i guess you'd need to check the env with raw ruby `ENV[]` style.

this also should fix tests, i hope!

Reply:
pushed, thanks.

Reply:
Nice catch @hjanuschka 👍